machAeroTutorials/airfoilopt_multipoint.rst

```diff
@@ -11,10 +11,26 @@ Optimization does not have to be limited to a single flight condition.
 This section goes through the same optimization as the single point case, except with one more flight condition. 
 Instead of rewriting the code from scratch, the differences in code will be pointed out.
 
+.. note::
```

Comment:
Is this a choice made to be sure that the scripts can run on machines with a few procs?

Reply:
No, not really, I think the main reason was to minimize diff from the single point, but @nwu63 can clarify. We discussed using the same strategy as we usually do, but its also covered later (in the wing opt).